frontend/src/types/activity.ts

```diff
@@ -46,6 +47,8 @@ export function activityName(type: ActivityType): string {
       return "Update issue task status";
     case "bb.pipeline.task.file.commit":
       return "Commit file";
+    case "bb.pipeline.task.statement.update":
+      return "Statement update";
```

Comment:
SQL update.

yeah, it's a bit weird, because on the model layer, we don't want it to be coupled with SQL. But for now, SQL is the only scenario, so we'd surface SQL on UI.